public/docs/ts/latest/glossary.jade

```diff
@@ -31,23 +31,22 @@ block includes
     ## Ahead of Time (AOT) Compilation
   .l-sub-section
     :marked
-      Angular applications can be compiled by developers at build-time.
+      You can compile Angular applications at build-time.
       By compiling your application using the compiler-cli, `ngc`, you can bootstrap directly
-      to a Module Factory, meaning you don't need to include the Angular compiler in your javascript bundle.
+      to a Module Factory, meaning you don't need to include the Angular compiler in your Javascript bundle.
       Ahead of Time compiled applications also benefit from decreased load time and increased performance.
 
   :marked
     ## Angular Module
   .l-sub-section
     :marked
-      Helps us organize an application into cohesive blocks of functionality.
-      An Angular module identifies the components, directives, and pipes that are used by the application
-      along with the list of external Angular modules that the application needs, such as `FormsModule`.
+      Helps you organize an application into cohesive blocks of functionality.
+      An Angular module identifies the components, directives, and pipes that the application uses      along with the list of external Angular modules that the application needs, such as `FormsModule`.
```

Comment:
Ah, these spaces didn't show in the jade file (looked like a hard return). Fixed!